changelog/fragments/kb-800fdeec6e5c.yaml

```diff
@@ -0,0 +1,72 @@
+      ([More info](https://github.com/kubernetes-sigs/kubebuilder/pull/2700)). Important: You might be affect from Kubernetes 1.25 if you do not ensure that the containers
+      used in your project are restrictive or are defined if the property labels `pod-security.kubernetes.io` on clusters ([More info](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)). Therefore, you suggest
+      you begin to configure your projects accordingly. ([More info](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/))
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
+    migration:
+      header: For Golang/Ansible/Helm/HybridHelm language-based operators (go/v3, ansible/v1, helm/v1, hybrid.helm/v1-alpha)
+      body: |
+        Following the steps
+        1) In `config/kdefault/manager_auth_proxy_patch.go` ensure the following security context:
+
+        ```yaml
+        ...
+        spec:
+          containers:
+          - name: kube-rbac-proxy
+            securityContext:
+              allowPrivilegeEscalation: false
+              capabilities:
+                drop:
+                  - ALL
+        ...
+        ```
+
+        2) In the `config/manager/manager.yaml` ensure the following security context:
+
+        ```yaml
+        spec:
+          securityContext:
+            runAsNonRoot: true
+            seccompProfile:
+              type: RuntimeDefault
+          containers:
+            - command:
+                - /manager
+              image: controller:latest
+              name: manager
+              securityContext:
+                allowPrivilegeEscalation: false
+                capabilities:
+                  drop:
+                    - ALL
+        ```
+        3) Run make bundle tagert to ensure that the above changes are also applied to
+        your bundles.
+  - description: >
+      For Golang-based language (go/v2), fix issue introduced by removing the GO111MODULE=on from Dockerfile. ([More info](https://github.com/kubernetes-sigs/kubebuilder/pull/2678))
+
+      **IMPORTANT NOTE**: If you are using `go/v2` we highly recommend you upgrade your project to latest `go/v3` plugin (default operator-sdk scaffold). Bw aware that
+      `go/v2` is a plugin which was kept to ensure the backwards compatibility with old Kubebuilder
+      layout produced using its CLI versions < `3.0.0`. ([More info](https://book.kubebuilder.io/plugins/go-v2-plugin.html))
+      Therefore, we are unable so long to upgrade this plugin and solve tech-debts and it is
+      very likely it be deprecated by the Kubebuilder community soon. Note that projects built
+      with this plugin does not produce CRDs for example using v1 APIs so that cannot work on
```

Comment:
```suggestion
      with this plugin do not produce CRDs for example using v1 APIs so that cannot work on
```

Reply:
Just clarifying, do we mean to say that go/v2 plugins do not scaffold v1 CRDs. They support only v1beta1 which is deprecated and will be removed soon. Or am I getting it wrong?